docs/source/en/_toctree.yml

```diff
@@ -130,8 +130,6 @@
   title: Conceptual Guides
 - sections:
   - sections:
-    - local: api/models
-      title: Models
     - local: api/attnprocessor
       title: Attention Processor
     - local: api/diffusion_pipeline
```

Comment:
Does it make sense to also move this to API reference docs? I don't think they're suitable under conceptual guides. WDYT?

Reply:
I believe the `AttnProcessor` should already be under the API section no?